appdatabase/migrations/sql/0014_local_notifications_preferences.up.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE local_notifications_preferences (
```

Comment:
Empirically, I think that with a 3-dimensional table we can handle all types of notifications.